docs/source/docker_compose.rst

```diff
@@ -0,0 +1,132 @@
+Using Docker Compose
+====================
+
+Scylla-Monitor is container-based, using Docker Compose file allows you to simplify starting and stopping the monitoring stack.
+
+Prerequisite 
+------------
+
+Make sure you have `docker` and `docker-compose` installed.
+
+Setting Prometheus
+------------------
+
+Prometheus configuration file contains among others the IP address of the alertmanager and either the location
+of the `scylla_server.yml` file or a Consul IP address of Scylla-Manager, if Scylla-Manager is used for server provisioning.
+
+You can use `./prometheus-config.sh` to generate the file, for example: 
+
+.. code-block:: shell
+
+   ./prometheus-config.sh --compose
+   
+For production usage, It is advice that you will use an external directory for the Prometheus database, make sure to create one and
+update the docker-compose file accordingly (see the docker-compose example).  
+
+Setting Grafana Provisioning
+----------------------------
+
+Grafana reads its provisioning configuration from files, one for the data-source and one
```

Comment:
from which files

Reply:
If the files are explained below, we should link to the below sections. If this is some other file, we should either explain the location or link to where it is explained.

Reply:
It reads the configuration from files, i.e. the method of configuration is file